content/en/user-guide/integrations/testcontainers/index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Testcontainers"
+tags: [
+  "testcontainers",
+  "test",
+  "java",
+  "jvm",
+  "kotlin",
+  "go",
+  "dotnet"
+]
+categories: ["LocalStack Community", "LocalStack Pro"]
+date: 2023-02-11
+weight: 5
+description: >
+  Use Testcontainers with LocalStack
+aliases:
+  - /integrations/testcontainers/
+---
+
+<img src="testcontainers-logo.svg" width="600px" alt="Testcontainers logo">
+
+## Overview
+
+[Testcontainers](https://www.testcontainers.com/) is a library that helps you to run your
+tests against real dependencies.
+
+In this guide, you will learn how to use [Testcontainers](https://www.testcontainers.com/)
+with LocalStack.
+
+## Covered Topics
+
+* [Installing the Localstack module](#installing-the-localstack-module)
+* [Obtaining a LocalStack container](#obtaining-a-localstack-container)
+* [Configuring the AWS client](#configuring-the-aws-client)
+* [Useful Links](#useful-links)
+
+### Installing the Localstack module
+
+{{< tabpane >}}
+{{< tab header="NuGet" lang="shell">}}
+dotnet add package Testcontainers --version 2.4.0
+{{< /tab >}}
+{{< tab header="Go" lang="go">}}
+go get github.com/testcontainers/testcontainers-go/modules/localstack
+{{< /tab >}}
+{{< tab header="Java (Maven)" lang="xml">}}
+<dependency>
+    <groupId>org.testcontainers</groupId>
+    <artifactId>localstack</artifactId>
+    <version>1.17.6</version>
+    <scope>test</scope>
+</dependency>
+{{< /tab >}}
+{{< tab header="Java (Gradle)" lang="gradle">}}
+testImplementation 'org.testcontainers:localstack:1.17.6'
+{{< /tab >}}
+{{< /tabpane >}}
+
+### Obtaining a LocalStack container
+
+{{< tabpane >}}
+{{< tab header=".NET" lang="csharp">}}
+/* The current released version of Testcontainers for .NET does not include a
+LocalStack module. However, developers who want to use Testcontainers with
+LocalStack should not be discouraged, as the Testcontainers team has already
+developed a LocalStack module that will be included in the next release. In the
+meantime, developers can still use Testcontainers' generic builder to create a
+LocalStack container. */
+
+const string localStackImage = "localstack/localstack:1.4.0";
+
+const ushort localStackPort = 4566
+
+var localStackContainer = new ContainerBuilder()
+    .WithImage(localStackImage)
+    .WithPortBinding(localStackPort, true)
+    .WithWaitStrategy(Wait.ForUnixContainer().UntilHttpRequestIsSucceeded(request =>
+        request.ForPath("/_localstack/health").ForPort(localStackPort)))
+    .Build();
+
+await localStackContainer.StartAsync()
+    .ConfigureAwait(false);
+{{< /tab >}}
+{{< tab header="Go" lang="go">}}
+container, err := localstack.StartContainer(
+    ctx,
+    localstack.OverrideContainerRequest(testcontainers.ContainerRequest{
+        Env: map[string]string{
+            "AWS_ACCESS_KEY_ID":     accesskey,
+            "AWS_SECRET_ACCESS_KEY": secretkey,
+            "AWS_SESSION_TOKEN":     token,
```

Comment:
I am not sure if these are needed here, since they are not covered by LocalStack Docs 🤷